go.mod

```diff
@@ -28,11 +28,11 @@ require (
 	github.com/Abirdcfly/dupword v0.0.14 // indirect
 	github.com/Antonboom/errname v0.1.13 // indirect
 	github.com/Antonboom/nilnil v0.1.9 // indirect
-	github.com/Antonboom/testifylint v1.3.1 // indirect
+	github.com/Antonboom/testifylint v1.4.3 // indirect
```

Comment:
Some indirect dependencies were updated. I suspect you used the `-u` flag when doing `go get -u ...` - I would advice against using that flag, as we can't be sure that the indirect dependency is correctly following semver and not introducing breaking changes. For further explanation see [this](https://go.dev/wiki/Modules#:~:text=A%20common%20mistake%20is%20thinking%20go%20get%20%2Du%20foo%20solely%20gets%20the%20latest%20version%20of%20foo.%20In%20actuality%2C%20the%20%2Du%20in%20go%20get%20%2Du%20foo%20or%20go%20get%20%2Du%20foo%40latest%20means%20to%20also%20get%20the%20latest%20versions%20for%20all%20of%20the%20direct%20and%20indirect%20dependencies%20of%20foo.).

Reply:
@lovromazgon strange. I don't normally use the `-u` flag, so not sure what happened.

Reply:
I'd suggest doing `git checkout main -- go.mod` so we have the `go.mod` from `main` and then again `go get` the dependencies we actually want to update (probably just the protocol).

Reply:
Yeah, that fixed it. Not sure what bumped that dependency 🤷 . Anyways, it should be good now.